.github/workflows/dbld-images.yml

```diff
@@ -55,15 +50,15 @@ jobs:
           gh_export UPLOAD_IMAGES_INTERNAL
 
       - name: Log in to the Container registry
-        if: env.UPLOAD_IMAGES_INTERNAL == 'true'
+        if: always() && env.UPLOAD_IMAGES_INTERNAL == 'true'
```

Comment:
why do we need the ```always()``` portion of the conditional?

Reply:
I agree it looks weird, but this the GitHub actions way to allow a following step to run if the previous fails.
https://docs.github.com/en/actions/learn-github-actions/expressions#job-status-check-functions

As the documentation states, there is an implicit `success()` condition at each step:
> A default status check of success() is applied unless you include one of these functions.